main.go

```diff
@@ -23,7 +23,8 @@ var versionCommand = &cobra.Command{
 
 func init() {
 	rcommand.AddCommand(versionCommand)
-	rcommand.AddCommand(listCommand)
+	rcommand.AddCommand(countCommand)
```

Comment:
This change is unrelated to this PR.
Renaming should be done in a different PR.